openedx/core/djangoapps/course_groups/cohorts.py

```diff
@@ -119,7 +119,7 @@ def is_course_cohorted(course_key):
     Raises:
        Http404 if the course doesn't exist.
     """
-    return _get_course_cohort_settings(course_key).is_cohorted
+    return get_course_cohort_settings(course_key).is_cohorted
```

Comment:
I will add a new getter to get the id that I am using from the cohort settings and update this.  I need more than the provided getter.